diff --git a/node_modules/@types/react/index.d.ts b/node_modules/@types/react/index.d.ts
index 788ad15..1465059 100644
--- a/node_modules/@types/react/index.d.ts
+++ b/node_modules/@types/react/index.d.ts
@@ -134,8 +134,8 @@ declare namespace React {
         | ((
             props: P,
         ) => ReactNode | Promise<ReactNode>)
-        // constructor signature must match React.Component
-        | (new(props: P, context: any) => Component<any, any>);
+        // Broadened for React Native mixin-based class components
+        | (new(props: P, ...args: any[]) => any);
 
     /**
      * Created by {@link createRef}, or {@link useRef} when passed `null`.
@@ -4039,8 +4039,9 @@ declare namespace React {
         // TODO: Check impact of making React.ElementType<P = any> = React.JSXElementConstructor<P>
         type ElementType = string | React.JSXElementConstructor<any>;
         interface Element extends React.ReactElement<any, any> {}
-        interface ElementClass extends React.Component<any> {
-            render(): React.ReactNode;
+        interface ElementClass {
+            props: any;
+            render?(): React.ReactNode;
         }
         interface ElementAttributesProperty {
             props: {};
